docs/debian_ubuntu_packaging.md

```diff
@@ -0,0 +1,410 @@
+files are not installed in that directory. However a symbolic link is
+created in `/usr/share/docs/aeternity-node/docs` to point to the files
+in `/opt/aeternity/node/docs/`. This is required since the `install`
+file does not provide a robust way to separate or exclude directories.
+
+
+### links file
+
+*debian/pacakge_name.links* [[1]](https://www.debian.org/doc/manuals/maint-guide/dother.en.html#links)
+
+Create `debian/package-name.links` if symbolic links are required for
+some reason (purely aesthetic or for operation purposes).
+
+```
+var/log/aeternity/node opt/aeternity/node/log
+opt/aeternity/node/docs usr/share/doc/aeternity-node/docs
+```
+
+### changelog file
+
+*debian/changelog* [[1]](https://www.debian.org/doc/manuals/maint-guide/dreq.en.html#changelog) [[2]](https://www.debian.org/doc/debian-policy/ch-source.html#debian-changelog-debian-changelog)
+
+Create `debian/changelog` file. This one is essential to package
+building. Packaging policy requires this file to be present to track
+changes introduced in newer versions and requires a special format. It
+is not advisable to try to create it by hand. It is best to see `man
+dch` or `man debchange`.
+
+Example:
+
+```
+aeternity-node (1.1.0) unstable; urgency=medium
+
+  * Initial Debianization
+
+ -- Full Name <full.name@aeternity.com>  Tue, 18 Dec 2018 09:43:00 +0200
+```
+
+## Generating changelog file.
+
+
+[Debian New Maintainers' Guide](https://www.debian.org/doc/manuals/maint-guide/dreq.en.html#changelog)
+and
+[Debian Policy](https://www.debian.org/doc/debian-policy/ch-docs.html#changelog-files-and-release-notes)
+require an up to date changelog file.
+
+The best solution would be to add the changes for every Aeternity
+release to the changelog file.  However there is no easy way to do it
+from existing data, as the changelog file has a special format.
+
+One way would be to use Git commit messages and the `git-buildpackage`
+tools. However this is not very practical for Aeternity codebase. This
+would require special branches in the Git repository and this is
+something that (potentially) should be avoided.
+
+Another approach would be to convert the RELEASE-NOTES file for every
+release into a Debian changelog file. However this is impractical. The
+RELEASE-NOTES file is a Markdown file with vague formatting (sections,
+topics etc.) and can't be converted to a Debian changelog file
+straight-forward.
+
+The simplest solution is to add a single comment in the Debian
+changelog file pointing to the RELEASE-NOTES file on disk (in the
+package) for the release. Even it might not be strictly in sync with
+the Debian changelog file policy. This is the approach used.
+
+### Using dch
+
+```
+cd ae_node_source/
+DEBEMAIL=info@aeternity.com
+DEBFULLNAME="Aeternity Team"
+AE_VERSION=$(cat VERSION)
+AE_VERSION_NOTE="Release notes are available in /usr/share/doc/aeternity-node/docs/release-notes/RELEASENOTES-$AE_VERSION.md"
```

Comment:
```suggestion
AE_VERSION_NOTE="Release notes are available in /usr/share/doc/aeternity-node/docs/release-notes/RELEASE-NOTES-$AE_VERSION.md"
```

Reply:
Fixed in 5142342. The whole example is improved in c409e5e. In sync with the commands used in the main Makefile (`$(DEB_PKG_CHANGELOG_FILE):`). Required after changes in the logic for the changelog file generation.